index.bs

```diff
@@ -647,17 +664,23 @@ Native depth sensing {#native-depth-sensing-section}
 
 Depth sensing specification assumes that the native device on top of which the depth sensing API is implemented provides a way to query device's <dfn>native depth sensing</dfn> capabilities. The device is said to support querying device's native depth sensing capabilities if it exposes a way of obtainig [=view=]-aligned depth buffer data. The depth buffer data MUST contain buffer dimensions, the information about the units used for the values stored in the buffer, and a <dfn>depth coordinates transformation matrix</dfn> that performs a coordinate system change from normalized view coordinates to normalized depth buffer coordinates. This transform should leave <code>z</code> coordinate of the transformed 3D vector unaffected.
 
+The device can <dfn>support depth sensing type</dfn> in 2 ways. If the device simply returns estimated depth values with minimal post-processing it is said to support {{XRDepthType/"raw"}} depth type. If the device or runtime can apply additional processing to "smooth" out noise from this data (e.g. into larger regions of the same depth value), it is said to support {{XRDepthType/"smooth"}} depth type.
```

Comment:
Nit: "with minimal post-processing it is said to support" is missing a comma before "it is said".

Nit: 'If the device or runtime can apply additional processing to "smooth" out noise from this data' - why is "smooth" in quotation marks here? Maybe we need a better word to describe what is happening here if quotes are needed.

Reply:
I had put quotes here because it was also the word that we are defining, which I think is somewhat of a convention. I would prefer not to use the word "smooth" here, but I don't know a good alternative. Open to suggestions.